packages/validator/test/e2e/slashing-protection-interchange-tests/tests/tests/generated/duplicate_pubkey_not_slashable.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
I don't think you want to commit this files to the source code, but download them on demand when testing

Reply:
so should we maybe delete them when the tests are done?  or just add them to the .gitignore?